cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -53,15 +67,58 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
           resultTable()
         }
       }
+      section("Changing the logged in users password", "administration-security-users-alter-password") {
+        p("Users can be change their own password using `ALTER CURRENT USER SET PASSWORD`, " +
+          "the old password is required in addition to the new. " +
+          "This will both change the password and set the `CHANGE NOT REQUIRED` flag.")
+        query("ALTER CURRENT USER SET PASSWORD FROM 'abc123' TO '123xyz'", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          // Can't pull on the result since the test is run with auth disabled, which is not allowed for this command
```

Comment:
Are all system commands run with auth-disabled in the docs tests? Perhaps we need to fix that?

Reply:
I think they are, and the show commands does not show any default users/roles/privileges